plugins/dataTypes/Boolean/README.md

```diff
@@ -0,0 +1,44 @@
+## AlphaNumeric Data Type
+
```

Comment:
This whole file needs to be updated. If you're not too sure on how the API works, no worries - I can write up the blurb for this file.